src/components/SelectAutocomplete/SelectAutocomplete.tsx

```diff
@@ -45,9 +45,7 @@ export function SelectAutocomplete({
       Select a currency
     </span>
   );
-  const [selected, setSelected] = React.useState<string>(
-    toString(hostProps.value ?? hostProps.defaultValue)
-  );
```

Comment:
This was odd to have and resulted in weird behaviour where the caller needs to change the selected coin. 

Given that the caller is passing in the `hostProps.value` this makes this state redundant and confusing.